src/services/test-helpers/mock/mockApi.ts

```diff
@@ -517,6 +519,151 @@ const auctionsWinningsAt = () =>
 		return optionWinnings;
 	});
 
+/**
+ * Asset specific constants.
+ * Note: It borrows some variables used in the parachains constant section
+ *
+ * Used in `/assets` and `/accounts` endpoints
+ */
```

Comment:
Anyone have an opinion on if we should break some fo the below out into a new file for organization?

Reply:
I was thinking the same thing. The file itself is starting to get bloated with endpoint specific functions. I am onboard for moving them out.

Reply:
I'm fine either way but I don't mind long test files generally. They are typically not read top to bottom anyway?